docs/en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -122,6 +122,17 @@ automatically joined as required:
  * `MyRecordSubclass_Live` table: Contains only live data for subclass columns
  * `MyRecordSubclass_Versions` table: Contains only version history for subclass columns
 
+While not explicit DataObjects, `many_many` relationships create their own sets of records on their own tables.
+These records represent content changes to a DataObject, and are therefore versioned.
+If you have, for instance, a versioned `Product` DataObject with `many_many` categories, the following tables will be created:
+
+* `Product`
+* `Product_Live`
+* `Product_versions`
+* `Product_Categories`
+* `Product_Categories_Live`
+* `Product_Categories_versions`
```

Comment:
This is the opposite to what is true.... many_many is NOT automatically versioned. To get many_many versioning you MUST use the `through` form of many_many and apply versioning to that intermediary dataobject class.